getsolus/packages
# baselayout: Switch to ypkg format

**Summary**

- Comar stuff isn't needed and will be dropped in eopkg py3
- Use `permanent` so eopkg never deletes the paths

**Test Plan**

- Install this pkg and canonical baselayout to a new rootfs and confirm the directory layouts are the same
- Create a new rootfs from this package:
  ```
  $ sudo /usr/bin/eopkg it baselayout-1.9.0-76-1-x86_64.eopkg -D /tmp/baselayout-rootfs/
  $ sudo /usr/bin/eopkg ur -D /tmp/baselayout-rootfs
  $ sudo /usr/bin/eopkg --ignore-comar it -c system.base -D /tmp/baselayout-rootfs
  ```
- Chroot in to a new solus rootfs created from this package:
  `sudo systemd-nspawn -D /tmp/baselayout-rootfs -b`
- Install package to a VM, reboot, install and remove a few packages and confirm nothing blows up.
- Install package locally, install updates, confirm nothing blows up

**Packager's Note**
- CI expected to fail and it doesn't understand package conversions

**Checklist**
